packages/core-components/src/layout/HeaderLabel/HeaderLabel.tsx

```diff
@@ -30,7 +29,7 @@ const useStyles = makeStyles(theme => ({
     letterSpacing: 0,
     fontSize: 14,
     height: '16px',
-    marginBottom: 2,
+    marginBottom: 4,
```

Comment:
Any reason why we're not using `rem` or `em` units throughout this project instead?

Reply:
No not really - but still good to nudge this a bit here. Was wondering about the static styles & adjusted theme to use the theme :)